composer.json

```diff
@@ -9,13 +9,14 @@
 	},
 	"require": {
 		"ext-openssl": "*",
-		"automattic/jetpack-logo": ">=1.0"
+		"automattic/jetpack-logo": "1.0.0"
```

Comment:
In #12480 we're changing this to `@dev`, should allow you to install the package properly without a `version` in its composer.json.